### ISSUE TITLE
Ability, Item, and Move Effect prose entries have "XXX" placeholder values

### ISSUE THREAD

**Issue by cmmartti** [veekun/pokedex#258]
Some of the prose entries for abilities, items, and move effects are placeholder values. For example, in `ability_prose.csv`:

```
ability_id,local_language_id,short_effect,effect
233,9,XXX new effect for neuroforce,XXX new effect for neuroforce
```

**Comment by magical** [comment]
Patches welcome :)

**Comment by cmmartti** [comment]
Yeah, I planned on submitting a patch, but I created this issue in case I forgot. What are the preferred data sources? I can copy them manually from Bulbapedia, unless there are better places.

**Comment by magical** [comment]
It's good to look at a variety of sources. Smogon is generally pretty reliable for move effects. Bulbapedia is okay too, though take anything they say with a grain of salt. Manual testing may be in order if you find contradictory sources.

cc @eevee, who wrote most of the original effects.
